src/templates/template-vision-segmentation/README.md

```diff
@@ -77,7 +77,7 @@ python main.py  \
   --node_rank 0 \
   --master_addr #:::= it.master_addr :::# \
   --master_port #:::= it.master_port :::# \
-  --backend nccl
+  --backend #:::= backend :::#
```

Comment:
In line 58 whenever, I am trying to replace the nccl string with `#:::= backend :::#`, the application is failing and I couldn't figure out why.